app/models/GetEstateSchemaModels.scala

```diff
@@ -55,13 +56,29 @@ object EstatePerRepOrgType {
 case class EstateWillType(name: NameType,
                           dateOfBirth: Option[LocalDate],
                           dateOfDeath: LocalDate,
-                          identification: Option[IdentificationType],
+                          identification: IdentificationType,
```

Comment:
Is the deceased person always going to have an identification in the response from etmp?